src/api/add.js

```diff
@@ -5,6 +5,23 @@ const addToDagNodesTransform = require('../add-to-dagnode-transform')
 const promisify = require('promisify-es6')
 
 module.exports = (send) => {
+  /**
+   * Add content to IPFS.
+   *
+   * @alias add
+   * @method
+   * @param {(Buffer|Stream|Array<Buffer|Stream>)} files - The content to add.
+   * @param {function(Error, {hash: string})} [callback]
+   * @returns {Promise<{hash: string}>|undefined}
+   *
+   * @memberof Api#
+   *
+   * @example
```

Comment:
As I understand our previous discussions, examples should make use of callbacks, not promises.